docs/en/ingest-management/elastic-agent/configuration/outputs/output-elasticsearch.asciidoc

```diff
@@ -295,6 +297,27 @@ options. Based on this configuration, the name would be:
 
 include::../authentication/kerberos-shared-settings.asciidoc[tag=kerberos-all-settings]
 
+[[output-elasticsearch-compatibility-setting]]
+=== Compatibility setting
+
+[cols="2*<a"]
+|===
+| Setting | Description
+
+// tag::allow_older_versions-setting[]
+|
+[id="{type}-allow_older_versions-setting"]
+`allow_older_versions`
+
+| Allow {agent} to connect and send output to an {es} instance that is at a version lower than the agent.
```

Comment:
```suggestion
| Allow {agent} to connect and send output to an {es} instance that running an earlier version than the agent.
```

Reply:
Alternate suggestion: 
"Allow {agent} to connect and send output to an {es} instance that is 
running a version earlier than the agent version. "

Reply:
Thanks. Changed to `| Allow {agent} to connect and send output to an {es} instance that is running an earlier version than the agent version.`